systemtest/src/main/java/io/strimzi/systemtest/resources/crd/kafkaclients/KafkaBasicExampleAdminClient.java

```diff
@@ -0,0 +1,258 @@
+        if (builder.topicOperation == null || builder.topicOperation.isEmpty())
+            throw new InvalidParameterException("TopicOperation must be set.");
+        if (builder.bootstrapAddress == null || builder.bootstrapAddress.isEmpty())
+            throw new InvalidParameterException("Bootstrap server is not set.");
+        if ((builder.topicName == null || builder.topicName.isEmpty()) && !(builder.topicOperation.equals("help") || builder.topicOperation.equals("list")))
+            throw new InvalidParameterException("Topic name (or 'prefix' if topic count > 1) is not set.");
+
+        replicationFactor = (builder.replicationFactor == 0) ? 1 : builder.replicationFactor;
+        partitions = (builder.partitions == 0) ? 1 : builder.partitions;
+        topicCount = (builder.topicCount == 0) ? 1 : builder.topicCount;
+        topicOffset = builder.topicOffset;
+        adminName = builder.adminName;
+        bootstrapAddress = builder.bootstrapAddress;
+        topicName = builder.topicName;
+        additionalConfig = builder.additionalConfig;
+        namespaceName = builder.namespaceName;
+        topicOperation = builder.topicOperation;
+    }
+
+    public String getAdminName() {
+        return adminName;
+    }
+
+    public String getBootstrapAddress() {
+        return bootstrapAddress;
+    }
+
+    public String getTopicName() {
+        return topicName;
+    }
+
+    public int getTopicCount() {
+        return topicCount;
+    }
+
+    public int getTopicOffset() {
+        return topicOffset;
+    }
+
+    public String getAdditionalConfig() {
+        return additionalConfig;
+    }
+
+    public String getNamespaceName() {
+        return namespaceName;
+    }
+
+    public int getPartitions() {
+        return partitions;
+    }
+
+    public int getReplicationFactor() {
+        return replicationFactor;
+    }
+
+    public String getTopicOperation() {
+        return topicOperation;
+    }
+
+    protected KafkaBasicExampleAdminClient.Builder newBuilder() {
+        return new KafkaBasicExampleAdminClient.Builder();
+    }
+
+    protected KafkaBasicExampleAdminClient.Builder updateBuilder(KafkaBasicExampleAdminClient.Builder builder) {
+        return builder
+                .withAdditionalConfig(getAdditionalConfig())
+                .withBootstrapAddress(getBootstrapAddress())
+                .withTopicOperation(getTopicOperation())
+                .withTopicCount(getTopicCount())
+                .withTopicOffset(getTopicOffset())
+                .withPartitions(getPartitions())
+                .withReplicationFactor(getReplicationFactor())
+                .withAdminName(getAdminName())
+                .withTopicName(getTopicName())
+                .withNamespaceName(getNamespaceName());
+    }
+
+    public KafkaBasicExampleAdminClient.Builder toBuilder() {
+        return updateBuilder(newBuilder());
+    }
+
+    public JobBuilder adminStrimzi() {
```

Comment:
Same as below, the name seems to be weird ... but also, why is this method really needed? It does exactly the same as `defaultAdminStrimzi`.

Reply:
@see-quick @Frawless or @im-konge might know better. I was just following the suit with this.

Reply:
Maybe that was some encapsulation -> just keep one of them

Reply:
Removed and using only renamed `defaultAdmin()`